openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -251,6 +251,12 @@ def cleanup_placeholder(self, placeholder, failed):
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        reset_frame_range(
+            playback=False,
+            render=False,
+            fps=False,
+            instances=True
+        )
```

Comment:
With these arguments I'd say at this stage you're just better off calling `lib.update_instances_frame_range()` to ignore the settings for reset frame range. Right?

Reply:
Good point. Lets go through `update_instances_frame_range`